src/lib/modules/scaffolding/description.ts

```diff
@@ -0,0 +1,106 @@
+import Ajv from "ajv";
+
+import { Logger } from "../../../typings/logger";
+import { schema } from "./schema";
+
+const fs = require("../../core/fs");
+
+const ajv = new Ajv();
+const scaffoldingSchema = ajv.compile(schema);
+
+interface Properties {
```

Comment:
Would be great if we could rename `Properties` to something less generic as well

Reply:
This is private type, it has only sense in this context os the recipe